node/metrics/src/names.rs

```diff
@@ -53,13 +58,18 @@ pub mod bft {
 pub mod blocks {
     pub const HEIGHT: &str = "snarkos_blocks_height_total";
     pub const TRANSACTIONS: &str = "snarkos_blocks_transactions_total";
+    pub const TRANSMISSIONS: &str = "snarkos_blocks_transmissions_total";
+    pub const SOLUTIONS: &str = "snarkos_blocks_solutions_total";
 }
 
 pub mod consensus {
     pub const CERTIFICATE_COMMIT_LATENCY: &str = "snarkos_consensus_certificate_commit_latency_secs";
     pub const COMMITTED_CERTIFICATES: &str = "snarkos_consensus_committed_certificates_total";
     pub const LAST_COMMITTED_ROUND: &str = "snarkos_consensus_last_committed_round";
     pub const BLOCK_LATENCY: &str = "snarkos_consensus_block_latency_secs";
+    pub const TRANSACTIONS: &str = "snarkos_consensus_transactions_total";
```

Comment:
should we maybe add unconfirmed to the consensus txn metric names just so its clear?

Reply:
Seems useful to me :+1:

Reply:
Sure!